29.md

```diff
@@ -40,7 +40,7 @@ Relays should prevent late publication (messages published now with a timestamp
 
 - *text root note* (`kind:11`)
 
-This is the basic unit of a "microblog" root text note sent to a group.
```

Comment:
not sure about this suggestion of not mixing `kind:{9.10}` with `kind:{11,12}` -- would be normal to have a chat (kind 9/10) about a post within the group so it wouldn't be that weird 🤷‍♂️

Reply:
Yeah potentially. I guess it's hard to tell without putting into action. I'm okay with removing that suggestion. Feel free to update it if you'd like. The important part was changing from microblog to forum.